investigator/investigator/advise_justification/investigate_advise_justification.py

```diff
@@ -24,14 +24,15 @@
 from .metrics_advise_justification import advise_justification_success
 from .metrics_advise_justification import advise_justification_in_progress
 from .metrics_advise_justification import advise_justification_type_number
+from prometheus_async.aio import track_inprogress, count_exceptions
 
 _LOGGER = logging.getLogger(__name__)
 
 DEPLOYMENT_NAME = os.environ["THOTH_DEPLOYMENT_NAME"]
 
 
-@advise_justification_exceptions.count_exceptions()
-@advise_justification_in_progress.track_inprogress()
+@count_exceptions(advise_justification_exceptions)
```

Comment:
we do not need one metric file per agent in this case then, we can define one metric common to all and use labels, WDYT?

Reply:
That's what we do. The individual metrics files just set the labels

Reply:
I thought we do not need those files anymore now as you can assign it directly in the main file. then LGTM